### DISCUSSION TITLE
How to enforce moving content to next line?

### DISCUSSION THREAD

**Discussion by eugenevk** [quasarframework/quasar#12346] [General - Components / Directives / etc] [answered]
I am using the flex 12-col grid to position QInput fields. How can I enforce a line break after a particular column to move the following content to the next line?

**Comment by MilosPaunovic** [comment, marked as answer]
Is there some project constraint that prevents you from creating a new row and placing the rest of elements there?

**Reply by eugenevk** [reply]
How do I do that @MilosPaunovic? Adding a new ```<div>```?

**Reply by MilosPaunovic** [reply]
```html
<div class="row">
  <div class="col-3">
    <!-- Input 1 -->
  </div>
  <div class="col-3">
    <!-- Input 2 -->
  </div>
</div>

<div class="row">
  <div class="col-3">
    <!-- Input 3 -->
  </div>
  <div class="col-4">
    <!-- Input 4-->
  </div>
</div>
```

**Reply by eugenevk** [reply]
Great, thanks @MilosPaunovic
